### PR TITLE
Document how to create a library.

### PR DESCRIPTION
**Pull request by Kaligule**
There is a link to a gif, taken from here:
https://github.com/excalidraw/excalidraw-libraries/pull/75

That pull request is also proof that this one is needed.

### CODE REVIEW

**Comment by YouriTjang** [comment]
I was about to create this documentation, but good to see you did it already 👍 
I also found [this pre-existing issue](https://github.com/excalidraw/excalidraw-libraries/issues/72)

File: README.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+3. Add each object you want in your library like so:
+  - Select the object
+  - Open the library menu (by clicking on the "library" button or
+    typing `9`)
+  - The selected object should appear in a square. Click on it (you
+    will see a `+` sign when hovering over it)
+4. Still in the library menu, click on "Export". You will get a
+   `.excalidrawlib` file with all your added objects in it.
+
+![Gif showing how to add a single objects and export the library](https://user-images.githubusercontent.com/1197663/111036434-69464e80-841f-11eb-8e3d-9a1c7dab5079.gif)
```
Comment (YouriTjang) [inline README.md:19]: Probably better to add the file in the repo here. Under /public, for example?
And a typo: objects, should be singular here.

Comment (Kaligule) [reply README.md:19]: I wasn't sure where the file should go. I will create a public dirctory.

About the singular: At this point we are talking about multiple objects, aren't we? The "for each object" loop is only in point 3

Comment (YouriTjang) [reply README.md]: Yeah i mean specifically in the line:

> Gif showing how to add a single objects and export the library

it should be 'object', singular

Comment (Kaligule) [reply README.md]: Ahh, I thought you were refering to point 4. Will fix.

Comment (Kaligule) [reply README.md]: Did so :)

**Comment by lipis** [comment]
Looks good! Thank you.

**Pull request merged**